kata-deploy/scripts/kata-deploy.sh

```diff
@@ -45,6 +51,20 @@ function install_artifacts() {
 	chmod +x /opt/kata/bin/*
 }
 
+function restart_cri_runtime() {
+	if [ "$1" == "containerd-k3s" ]; then
+		if systemctl list-units | grep -q "k3s-agent"; then
+			# worker node
+			systemctl restart k3s-agent && sleep 10
+		else
+			# master node
+			systemctl restart k3s && sleep 10
```

Comment:
As is, this could be problematic for single master, multi-node clusters since the master server has to restart, potentially causing a simultaneous `kubectl describe/label ...` call on a worker node to fail.

Reply:
Thanks for raising @wilsonianb but I'm not comfortable with sleeps like this. Isn't the service "ready" once the `systemctl restart` returns? If not, I think it would be better to check the service status (or some other method) for the expected state periodically, and timeout/die after a much longer timespan.

Reply:
Removed the `sleep`s and now it seems to actually successfully run on two node (master + worker) clusters :upside_down_face: